src/igvfd/schemas/workflow.json

```diff
@@ -0,0 +1,85 @@
+{
+    "title": "Workflow",
+    "$id": "/profiles/workflow.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submitting a worfklow.",
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "source_url",
+        "name"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "accession",
+        "alternate_accessions",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/references"
+        }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "accession": {
+            "accessionType": "WF"
+        },
+        "name": {
+            "title": "Name",
+            "description": "The preferred viewable name of the workflow.",
+            "type": "string",
+            "pattern": "^(\\S+(\\s|\\S)*\\S+|\\S)$"
+        },
+        "source_url": {
+            "title": "Source URL",
+            "description": "An external resource to the code base of the workflow in github.",
+            "type": "string",
+            "format": "uri",
+            "pattern": "^https?://github\\.com/(\\S+)$"
+        },
+        "workflow_repositories": {
+            "title": "Workflow Repositories",
+            "description": "Repository links for the workflow other than the code base in github.",
```

Comment:
description: "Resources hosting the workflow."
comment: add tot he existing comment as an example Dockstore